src/test/java/org/llorllale/cactoos/matchers/NotEmptyTest.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
@dgroup same here, this belongs to a different PR

Reply:
Done.